tdrs-backend/tdpservice/search_indexes/parsers/test/test_tanf_validators.py

```diff
@@ -0,0 +1,218 @@
+"""Test preparser functions and tanf_validator."""
+import pytest
+
+from tdpservice.search_indexes.models import T1
+from tdpservice.search_indexes.parsers.tanf_validators import (
```

Comment:
Might be better to just `import *` here.

Reply:
I didn't import * because it's been frowned upon at other positions. I think it's because it imports functions you might never even use. 

Maybe in the future we split the validators up into separate files?